Branch-SDK/src/io/branch/referral/ServerRequestQueue.java

```diff
@@ -286,8 +286,6 @@ public void moveInstallOrOpenToFront(String tag, int networkCount) {
                         || req.getTag().equals(BranchRemoteInterface.REQ_TAG_REGISTER_OPEN))) {
                     if (req.getTag().equals(BranchRemoteInterface.REQ_TAG_REGISTER_INSTALL)) {
```

Comment:
We should honor the incoming tag (method argument) and remove the entire if/else block.